fulp_modules/ZFulpcode/modules/jobs/inventories.dm

```diff
@@ -1,13 +1,13 @@
 /// Security (Original by Surrealistik) ///
 
 /datum/outfit/job/security
-	backpack_contents = list() //They start out with a crowbar!
-	box = /obj/item/storage/box/survival/security/improved
+	backpack_contents = list(/obj/item/melee/baton/loaded=1) //we want to keep the stuff they had before
```

Comment:
backpack_contents = list(/obj/item/melee/baton/loaded=1) //we want to keep the stuff they had before
```suggestion

```
This line will overwrite any new item added to backpacks and will need to be updated, it also simply doesn't need to exist.